internal/provider/models/alerts/base_model.go

```diff
@@ -375,15 +380,59 @@ func GetAlertPayloadFromModel(v attr.Value, dd *diag.Diagnostics) map[string]any
 		"name": serviceName,
 	}
 
-	if serviceName != "log_analysis" && serviceValues["uri"].(StringValue).IsNull() {
+	// Check that there are no extra properties that the service doesn't expect
+	for attribute := range serviceValues {
```

Comment:
🤮

Reply:
So I tried my best to get validation working with the TF framework, but retrieving a *value* to validate against is an elusive topic.  Given that TF docs are horrible, it was easier to do a manual check here.

Reply:
I mean i assume this was the "best" way to do it for "reasons" it just makes me sad lol

Reply:
fixed.  Reworded errors to make more sense in the specific use case.